modules/menu/menu.admin.inc

```diff
@@ -42,7 +42,7 @@ function menu_overview_form(&$form_state, $menu) {
   menu_tree_check_access($tree, $node_links);
   $menu_admin = FALSE;
 
-  $form = _menu_overview_tree_form($tree);
+  $form = _menu_overview_tree_form($tree, $form_state);
```

Comment:
Interesting...

If your goal is for the code to be less misleading, I'd advocate for the reverse change instead: remove $form_state completely from _menu_overview_tree_form(). This is only called early on form rebuild, when $form_state['values'] is never filled, so it doesn't do anything.

As a bonus, that also makes sure there's no behavior change.

(Not that I think it's necessary per se - I have no strong opinion on dead D6 code that doesn't do anything.)